packages/redux-routine/src/runtime.js

```diff
@@ -29,7 +28,7 @@ export default function createRuntime( controls = {}, dispatch ) {
 			// Async control routine awaits resolution.
 			routine.then(
 				yieldNext,
-				( error ) => yieldError( castError( error ) ),
+				( error ) => yieldError( error ),
```

Comment:
I think this can be simplified even further.

```suggestion
				yieldError,
```

At which point, we could probably just collapse it all into a single line:

```js
routine.then( yieldNext, yieldError );
```